### PR TITLE
Update the Repo README & several charts README

### PR DESCRIPTION
**Pull request by ericsyh**
*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #<xyz>

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

Since some charts have been migrated and deprecated, we should update the related README to eliminate confusion. 

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [ ] `no-need-doc` 
  
  (Please explain why)
  
- [x] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,13 +27,12 @@ This repository includes the officially supported Helm Charts.
...
+| [`function-mesh-operator`(Migrated)](https://github.com/streamnative/charts/tree/master/charts/function-mesh-operator) | The officially supported Helm Chart for installing FunctionMesh to manage Pulsar Functions and Pulsar IO connectors on Kubernetes. | For details about how to use it, see [here](charts/function-mesh-operator/README.md). | 
+| [`pulsar`(Deprecated)](https://github.com/streamnative/charts/tree/master/charts/pulsar) | The officially supported Helm Chart used for deploying Apache Pulsar to Kubernetes clusters. | For details about how to use it, see [here](charts/pulsar/README.md). |  
+| [`local-storage-provisioner`](https://github.com/streamnative/charts/tree/master/charts/local-storage-provisioner) | The officially supported Helm Chart for installing the local storage provisioner use local persistent volumes as the persistent storage. | For details about how to use it, see [here](charts/local-storage-provisioner/README.md). |
```
Comment (Huanli-Meng) [inline README.md:34]:
```suggestion
| [`local-storage-provisioner`](https://github.com/streamnative/charts/tree/master/charts/local-storage-provisioner) | The officially supported Helm Chart for installing the local storage provisioner to use local persistent volumes as persistent storage. | For details about how to use it, see [here](charts/local-storage-provisioner/README.md). |
```

File: README.md [hunk 2]
```diff
@@ -27,13 +27,12 @@ This repository includes the officially supported Helm Charts.
...
+| [`image-puller`](https://github.com/streamnative/charts/tree/master/charts/image-puller) | The officially supported Helm Chart for installing an Image Puller which pulls docker images required for deploying Apache Pulsar or StreamNative Platform. | For details about how to use it, see [here](charts/image-puller/README.md). |
```
Comment (Huanli-Meng) [inline README.md:35]:
```suggestion
| [`image-puller`](https://github.com/streamnative/charts/tree/master/charts/image-puller) | The officially supported Helm Chart for installing an Image Puller which pulls required docker images for deploying Apache Pulsar or StreamNative Platform. | For details about how to use it, see [here](charts/image-puller/README.md). |
```

File: charts/function-mesh-operator/README.md [hunk 3]
```diff
@@ -1,55 +1,5 @@
...
-
-## Uninstall FunctionMesh operator
-
-Use the following command to uninstall FunctionMesh operator.
-
-```bash
-helm uninstall function-mesh
-```
\ No newline at end of file
+You can read our complete [documentation](https://functionmesh.io).
```
Comment (Huanli-Meng) [inline charts/function-mesh-operator/README.md:5]:
```suggestion
For details about how to install and uninstall the Function Mesh operator, see [Function Mesh documentation](https://functionmesh.io/docs/install-function-mesh). 
```

File: charts/pulsar/README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 
-# Apache Pulsar Helm Chart
+# Apache Pulsar Helm Chart (Deprecated)
 
-> Note: Most of the changes of this chart have been contributed back to https://github.com/apache/pulsar-helm-chart
+> Note: This Apache Pulsar Helm Chart has been deprecated and we recommand to use [streamnative/sn-platform](https://github.com/streamnative/charts/tree/master/charts/sn-platform) to provision and manage the Apache Pulsar on Kubernetes.
```
Comment (Huanli-Meng) [inline charts/pulsar/README.md:4]:
```suggestion
> **Note**
>
> This Apache Pulsar Helm Chart has been deprecated and we recommend using [streamnative/sn-platform](https://github.com/streamnative/charts/tree/master/charts/sn-platform) to provision and manage the Apache Pulsar on Kubernetes.
```

File: charts/sn-platform/README.md [hunk 5]
```diff
@@ -1,311 +1,21 @@
...
-    - [x] Kafka-on-Pulsar
-    - [x] Presto
-- [x] StreamNative Control Center:
-    - [x] StreamNative Console
-    - [x] Node Exporter
-    - [x] Prometheus
-    - [x] Grafana
-    - [ ] Grafana Loki
-    - [x] Alert Manager
+- Deploy and manage Pulsar in your private cloud environment
```
Comment (Huanli-Meng) [inline charts/sn-platform/README.md:17]:
```suggestion
- Deploy and manage Pulsar in your private cloud environment.
```

File: charts/sn-platform/README.md [hunk 6]
```diff
@@ -1,311 +1,21 @@
...
+- Deploy and manage your platform as a cloud-native system on Kubernetes
```
Comment (Huanli-Meng) [inline charts/sn-platform/README.md:18]:
```suggestion
- Deploy and manage your platform as a cloud-native system on Kubernetes.
```

File: charts/sn-platform/README.md [hunk 7]
```diff
@@ -1,311 +1,21 @@
...
-To uninstall StreamNative Platform, execute these commands.
-
-```bash
-export NAMESPACE=KUBERNETES_NAMESPACE
-helm uninstall vault-operator -n $NAMESPACE
-helm uninstall cert-manager -n $NAMESPACE
-helm uninstall pulsar-operator -n $NAMESPACE
-helm uninstall function-mesh -n $NAMESPACE
-```
+You can read our complete [documentation](https://docs.streamnative.io/docs/platform-overview).
```
Comment (Huanli-Meng) [inline charts/sn-platform/README.md:21]:
```suggestion
For details, see [StreamNative Platform documentation](https://docs.streamnative.io/docs/platform-overview). 
```

**Review by Huanli-Meng — APPROVED** [review]
Leave some of my comments. PTAL. Thanks.

**Review by xiaogang-sn — DISMISSED** [review]

File: README.md [hunk 8]
```diff
@@ -27,13 +27,12 @@ This repository includes the officially supported Helm Charts.
...
-
+| [`pulsar-operator`](https://github.com/streamnative/charts/tree/master/charts/pulsar-operator)  | The officially supported Helm Chart for installing the Pulsar operator to provide Customer Resource Definitions (CRDs) and Controllers to manage and run Pulsar clusters in a more resilient way. | For details about how to use it, see [here](charts/pulsar-operator/README.md). |
+| [`sn-platform`](https://github.com/streamnative/charts/tree/master/charts/sn-platform) | The officially supported Helm Chart used for deploying StreamNative Platform to Kubernetes clusters. | For how to use it, see [here](charts/sn-platform/README.md).
+| [`function-mesh-operator`(Migrated)](https://github.com/streamnative/charts/tree/master/charts/function-mesh-operator) | The officially supported Helm Chart for installing FunctionMesh to manage Pulsar Functions and Pulsar IO connectors on Kubernetes. | For details about how to use it, see [here](charts/function-mesh-operator/README.md). |
```
[the remark this answers is not in the document]
Comment (ericsyh) [reply README.md:32]: oh, sure. We should use the function mesh repo here.

[the remark this answers is not in the document]
Comment (ericsyh) [reply README.md]: @Huanli-Meng Updated in https://github.com/streamnative/charts/pull/948/commits/2f909b743d449619bc7e3b9e2c6cb7f4ac2a72cb

**Review by Huanli-Meng — DISMISSED** [review]
LGTM

**Review by Huanli-Meng — APPROVED** [review]

**Review by tuteng — APPROVED** [review]

**Pull request merged**